content/community/community-team/engagement-channels/contents.lr

```diff
@@ -0,0 +1,173 @@
+_model: page
+---
+_template: page-with-toc.html
+---
+title: Community Team: Engagement Channels
+---
+description: This page serves as a guide for Community Team members on how to engage with CC Open Source across various channels.
+---
+is_long_page: true
+---
+body:
+
+You've joined the CC Community Team. Congratulations! This is a very clear
+indication of your desire to engage more deeply with the Creative Commons
+community. As a fully remote organisation, CC uses a lot of tools to stay
+organised and collaborate. As you join the Community Team, depending on your
+role, you will be granted different levels of access to these tools and
+software.
+
+Given the breadth and diversity of these tools coupled with varying levels of
+access, it can be easy to not know what is expected of you and how you can make
+the most of your involvement. This guide will condense this information down to
+help you identify the best ways to engage with the team. _Let's get started!_
+
+## Meetings
+
+We regularly hold meetings to discuss work and, occasionally, to play games.
+Based on your role and the project, you will be invited to some of these
+meetings. You can join the CC Engineering calendar to keep up with the schedule.
+
+<!-- TODO: describe meetings here as they are finalized -->
```

Comment:
I think there's just the monthly community meeting and Vocabulary sprint planning

Reply:
Added the meetings in 2fa68bb.